projects/scion/workbench/src/lib/routing/workbench-navigational-states.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2018-2022 Swiss Federal Railways
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+/**
+ * Keys for passing workbench-specific state to a navigation.
+ *
+ * State can be read from {@link ActivatedRoute.data} using the {@link WorkbenchRouteData.state} key.
+ */
+export namespace WorkbenchNavigationalStates {
+  /**
+   * Key for reading CSS class(es) from navigational state.
+   */
+  export const cssClass = 'ɵcssClass';
```

Comment:
Consider removing this namespace and just using `WorkbenchRouteData.cssClass` instead.
Of course, route data and state are two different things. But having to think about the `cssClass` being part of the state, which itself is part of the route data is quite confusing.

Reply:
As discussed, we leave the namespace as it is because it indicates the different uses.